summoner-cli/src/Summoner/Source.hs

```diff
@@ -20,19 +20,31 @@ import qualified Toml
 
 -- | Type of the source resource.
 data Source
-    -- | URL link to the source file.
+    {- | URL link to the source file. Such files will be downloaded by URL. But
+    they are ingored in the @offline@ mode.
+    -}
     = Url !Text
-    -- | File path to the local source file.
+
+    {- | File path to the local source file.
+    -}
     | File !FilePath
-    -- | Link to external file.
+
+    {- | Link to external file. Generated file contains some short text and a
+    a give url link.
+    -}
```

Comment:
Now when we have `Raw`, this one is unnecessary. Do you think we can get rid of it?

Reply:
@vrom911 Yes, I also got the same idea :slightly_smiling_face: `Link` constructor now becomes redundant because is a strict and less useful subset of `Raw` which gives more flexibility. I think it's okay to remove it now and just write a migration guide how to use `Raw` :slightly_smiling_face: